README.md

```diff
@@ -90,7 +90,6 @@ Deploying using Helm consists of 4 steps
 
 <details><summary>Install StackRox Central Services</summary>
 
-#### Default Central Installation
```

Comment:
nitpick: I agree that this header is not important and it's redundant. Still, I see two beneficial aspects keeping it:
- it generates a link in rendered markdown (someone can use that link to directly point to this section)
- rendered text is a bit nicer, because of added space around it

The same comment applies to `Default Secured Cluster Installation`.

Reply:
Thanks for the comment! I had removed the headers as part of an effort to improve the structure of the text after my changes, but I don't think there is actually a need to remove them altogether, it's enough to shuffle them a bit. What do you think?